Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_009.LinkedInCredentials.sarif

```diff
@@ -83,11 +83,10 @@
             }
           ],
           "fingerprints": {
-            "AssetFingerprint/v1": "[id=0000000aaaaaaa][platform=LinkedIn]",
-            "ValidationFingerprint/v1": "[id=0000000aaaaaaa][secret=00000000aaaaaaaa]",
-            "ValidationFingerprintHash/v1": "adf099a2a5d039c5923f6d8de7f6e96ee22bcb3b1674596df5ab22198a0bd8b4",
```

Comment:
>adf099a2a5d039c5923f6d8de7f6e96ee22bcb3b1674596df5ab22198a0bd8b4 [](http://example.com/codeflow?start=45&length=64)

I was not expecting this.
This means that we won't be able to link new results with old results